src/__Private/LintRunConfig.hack

```diff
@@ -240,6 +243,37 @@ final class LintRunConfig {
     );
   }
 
+  <<__Memoize>>
+  public function getLinterConfigForLinter<TLinter as BaseLinter, TConfig>(
+    classname<TLinter> $classname,
+  ): ?TConfig where TConfig = TLinter::TConfig {
+    $config = idx(Shapes::idx($this->configFile, 'linterConfigs'), $classname);
+
+    if ($config is null) {
+      // Test for TypeAssert unsupported types, even when no config is supplied.
+      // This makes it impossible to accidentally ship a linter with a TypeAssert
+      // type that is not supported (if never tested with a config).
+      try {
+        $classname::typeAssertConfig(null);
```

Comment:
[1] This could be done in a test instead of at runtime (everytime any linter is invoked).